index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <!-- <script defer src="script.js"></script> -->
+    <title>Document</title>
```

Comment:
Would be nicer to give your html a title, when you open it in a browser it says that title, instead of it just saying document